tensorboard/webapp/metrics/actions/index.ts

```diff
@@ -134,3 +134,17 @@ export const cardPinStateToggled = createAction(
   '[Metrics] Card Pin State Toggled',
   props<{cardId: CardId; canCreateNewPins: boolean; wasPinned: boolean}>()
 );
+
+export const timeSelectionChanged = createAction(
+  '[Metrics] Linked Time Selection Changed',
+  props<{
+    startStep: number;
+    startWallTime: number;
+    endStep?: number;
+    endWallTime?: number;
```

Comment:
curious why it's named 'wall' time?

Reply:
The term originates from https://en.wikipedia.org/wiki/Elapsed_real_time#:~:text=Elapsed%20real%20time%2C%20real%20time,at%20which%20the%20task%20started.

and in context of TensorBoard, the "Wall" option.
![image](https://user-images.githubusercontent.com/2547313/128060063-5c9e4b93-1b30-4042-b875-001394d6e670.png)